spec.md

```diff
@@ -15,4 +14,569 @@ The key words "unspecified", "undefined", and "implementation-defined" are to be
 An implementation is not compliant if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
 An implementation is compliant if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
 
+## Terminology
+
+| Term              | Definition                                       |
+|-------------------|--------------------------------------------------|
+| Bucket            | A flat organization of container that store files.                          |
+| COSI  System              | Container Orchestration system/ Container Object Storage Implementation, communicates with Plugins using COSI service RPCs.                                     |
```

Comment:
add:
`COSI Node Adapter (aka CSI Driver)   |  a process that accepts gRPC calls from the Kubelet notifying it when the workload has been started and when it is terminating.`

Reply:
done